internal/driver/macmapper_test.go

```diff
@@ -178,3 +182,232 @@ func TestMatchEndpointRefAddressToMAC(t *testing.T) {
 		})
 	}
 }
+
+func TestMACAddressMapper_UpdateMappings(t *testing.T) {
```

Comment:
Remove Underscore in name and add a comment to the test case above to be consistent with other unit tests.

Reply:
actually I think this is a common naming scheme, GoLand recommends these names itself. `Test<TypeName>_<FuncName>`